### PR TITLE
Remove dead code

### PR DESCRIPTION
**Pull request by simonrozsival**
This obsolete command is no longer needed.

Squash whenever

### CODE REVIEW

File: Toggl.Daneel/ViewControllers/EditTimeEntryViewController.cs [hunk 1]
```diff
@@ -137,7 +137,7 @@ public override void ViewDidLoad()
 
             bindingSet.Bind(StartDateTimeView)
```
Comment (kornelijepetak) [inline Toggl.Daneel/ViewControllers/EditTimeEntryViewController.cs]: Maybe it would be good to change the name of the view too?

Comment (willsb) [reply Toggl.Daneel/ViewControllers/EditTimeEntryViewController.cs]: Not good: Needed. MvvmCross resolves these things by convention

Comment (simonrozsival) [reply Toggl.Daneel/ViewControllers/EditTimeEntryViewController.cs]: I can do that but in the 1429 PR.

Comment (kornelijepetak) [reply Toggl.Daneel/ViewControllers/EditTimeEntryViewController.cs]: @willsb wait, what? What does MvvmX do with the view name exactly?

Comment (willsb) [reply Toggl.Daneel/ViewControllers/EditTimeEntryViewController.cs]: Check it yourself by clicking [here](https://github.com/MvvmCross/MvvmCross/search?utf8=%E2%9C%93&q=MvxPostfixAwareViewToViewModelNameMapping&type=)

**Pull request merged**